packages/lodestar/src/network/util.ts

```diff
@@ -73,58 +65,30 @@ export function encodeResponseChunk(config: IBeaconConfig, method: Method, {err,
       type = config.types.Goodbye;
       break;
     case Method.BeaconBlocksByRange:
+      type = config.types.BeaconBlocksByRangeRequest;
+      break;
     case Method.BeaconBlocksByRoot:
-      type = config.types.SignedBeaconBlock;
+      type = config.types.BeaconBlocksByRootRequest;
       break;
-    default:
-      throw new Error(`Unhandled method ${method}`);
   }
-  data = Buffer.from(type.serialize(output));
-  return encodeSingleChunk(data);
+  return type;
 }
 
-export function decodeResponseChunk(config: IBeaconConfig, method: Method, chunk: Buffer): ResponseChunk {
-  const code = chunk[0];
-  if (code !== 0) {
-    throw new Error(chunk.slice(1).toString("utf8"));
-  }
-  const length = varint.decode(chunk, 1);
-  const bytes = varint.decode.bytes;
-  if (
-    length !== chunk.length - (bytes + 1) ||
-    length > MAX_CHUNK_SIZE
-  ) {
-    throw new Error(ERR_INVALID_REQ);
-  }
-  const data = chunk.slice(bytes + 1);
-  let output: Status | Goodbye | SignedBeaconBlock;
+export function getResponseMethodSSZType(
```

Comment:
Can we make getRequestMethodSSZType and getResponseMethodSSZType a single function?
eg: getMethodSSZType

Reply:
like add extra param "isResponse"?

Reply:
I don't think we need to differentiate between request and response ssz types.
just smoosh the two `switch` statements together

Reply:
But we do need. BeaconBlocksByRangeRequest is reuqest type while SignedBeaconBlock is response ssz type

Reply:
they are completely different types

Reply:
Oh I see, yeah, ignore this comment xD